web/views/metrics.erb

```diff
@@ -1,50 +1,60 @@
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/3.9.1/chart.min.js"></script>
```

Comment:
We will need to vendor this. The Web UI should be totally self-contained.

Reply:
Make sure our copy of chart.js.min includes the MIT license in their repo.

Reply:
Oh, looks like it already does include a small license header.

Reply:
Scripts are now vendored.